fAbout.resx

```diff
@@ -1,2969 +1,2969 @@
-        WiCaxHRml9B+5ZTbifuYzHnXp5lIIgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAn7NgCpnHdcOU3Yv/lN6M/5TdjP+X0X/vm8JunaWYQT4AAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAALFtGAK0i1JstJhs/7Wcc/+0nHP/tJxy/7Wcc/+0m3L/tJtz/7Wcc/+1nHP/tZxy/7WZ
-        bf+1mGn/tJZj/7SVXv+zkln/rIlU/6R/UP+bdUv/kmtG/11BRf8lGEn/GRBL/xkQTP8ZEE//GRBR/xgQ
-        Uv8bElT/NTBx/0E9fv8fF1n/GBBR/xkQUv8ZEVL/KyRm/0VBg/8nH2H/GRBS/xkQUf8YEFD/GRBN/xkQ
-        S/8ZEEr/LyBI/4BZQP+Vbkj/nXhM/6aCUv+ujFb/tJNb/7WVYP+0l2X/tZhq/7Sab/+1nHL/tZxz/7Wc
-        c/+1nHP/tJty/7Wcc/+1nHP/tZxz/7Wcc/+0lGXZtI1YFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAKeMMRifsF1smMp405XZh/+T3Yz/ldaE/5+yXGQAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACkoFAAnbtpTpbUgumT3Yv/k92L/5Pdi/+U2Yf/ltB+35u7
-        aniepFEgsm0MBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAALOJVDC0l2vZtJty/7Wbcv+0nHL/tJty/7Sb
-        cv+1nHP/tZxz/7Wcc/+1nHL/tZlt/7SYaf+1lmP/tZVe/7OSWf+siVT/o39Q/5t1S/+Ta0b/iWFC/1xA
-        R/8aEU7/GRBQ/xgQUf8YEFH/GRBS/xoRU/8sJ2X/MjBs/x0VVv8YEFL/GRBR/xkQUv8kHl3/NzVw/yMc
-        XP8ZEFL/GBBS/xkQUf8YEFD/GBBP/ysdTP9zUUT/jWRC/5VuSP+deEz/poJR/66LVf+0k1v/tZVf/7WX
-        Zf+1mGr/tZpv/7Wbcv+1nHL/tZxz/7Wcc/+1m3L/tZxz/7Wccv+1nHL/tJtx/7SSYZGyfj0GAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAArnkgAqSgSw6duGhEl817vZbVg/eT3Yv/k96L/5Tdi/2aw2+5o6FKEgAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACueSAOmsJwlZLc
-        iv+Q2oj/jteF/43Vg/+L1IL/i9F+/467aduSplBslJxIEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
-        AAAA
```

Comment:
`Majora's`

Reply:
Please update "Maintained by the Majoras Mask Randomizer Community Discord." Majoras to Majora's